src/lesson1/L1Task4.java

```diff
@@ -0,0 +1,8 @@
+package lesson1;
+
+public class L1Task4 {
+    //Створити метод перетворення градусів фаренгейта в градуси цельсія
+    public static double convertToCelsius (double degreeFahrenheit) {
```

Comment:
Зайвий пробіл

Reply:
Я вибачаюсь, але чомусь не бачу. Можливо я не туди дивлюся. Можна уточити де він зайвий?

Reply:
між назвою методу і дужкою з параметрами метода пробіл не треба 
```suggestion
    public static double convertToCelsius(double degreeFahrenheit) {
```